pkg/client/mapiserver/utils.go

```diff
@@ -62,3 +76,89 @@ func ChangeDateFormatToUnixDate(snapshotDisks []SnapshotInfo) error {
 	}
 	return nil
 }
+
+// postRequest sends request to a url with payload of values
+func postRequest(url string, values []byte, namespace string, chkbody bool) ([]byte, error) {
+
+	req, err := http.NewRequest("POST", url, bytes.NewBuffer(values))
+
+	if err != nil {
+		return nil, err
+	}
+
+	req.Header.Add("Content-Type", "application/json")
+
+	if len(namespace) > 0 {
+		req.Header.Set("namespace", namespace)
+	}
+
+	c := &http.Client{
+		Timeout: volumeCreateTimeout,
+	}
+
+	resp, err := c.Do(req)
+
+	if err != nil {
+		return nil, err
+	}
+
+	defer resp.Body.Close()
+
+	body, err := ioutil.ReadAll(resp.Body)
+
+	if err != nil {
+		return nil, err
+	}
+
+	code := resp.StatusCode
+
+	if chkbody && err == nil && code != http.StatusOK {
+		return nil, fmt.Errorf(string(body))
+	}
+
+	if code != http.StatusOK {
+		return nil, fmt.Errorf("Server status error: %v", http.StatusText(code))
+	}
+
+	return nil, nil
+}
+
+// getRequest GETS a request to a url and returns the response
+func getRequest(url string, namespace string, chkbody bool) ([]byte, error) {
+
+	req, err := http.NewRequest("GET", url, nil)
+
+	if err != nil {
+		return nil, err
+	}
+
+	if len(namespace) > 0 {
+		req.Header.Set("namespace", namespace)
+	}
+
+	c := &http.Client{
+		Timeout: timeoutVolumeDelete,
```

Comment:
Can we have a single timeout constant .... `DefaultVolumeTimeout` in all volume related operations. ?

Reply:
yes we can have it